api/RTCPeerConnection.json

```diff
@@ -1600,32 +1348,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getStats",
           "spec_url": "https://w3c.github.io/webrtc-pc/#widl-RTCPeerConnection-getStats-Promise-RTCStatsReport--MediaStreamTrack-selector",
           "support": {
-            "chrome": [
-              {
-                "version_added": "58",
-                "notes": "Promise resolves with <code>RTCStatsReport</code>."
```

Comment:
What did this note mean? Are you sure it doesn't matter?

Reply:
After digging through the commit history, it looks like this comes from #3287, and is about https://chromestatus.com/feature/5665052275245056.  It seems to be a change in the object that's returned by `getStats()`, but honestly, I'm not seeing anything different between Chrome 57 and the latest Chrome (or Chrome 58 and 60 for that matter).

![Screen Shot 2021-09-08 at 02 39 40](https://user-images.githubusercontent.com/5179191/132485977-dc86babb-4934-4bb1-9afc-1a42272d607c.png)

I didn't think this note was that important, particularly for such an old Chrome version, but now it looks like it's incorrect as well.

Reply:
I think what happened here is that the promise-based variant has always returned something different from the callback-based variant. So a `returns_promise` subfeature should be enough here.